src/azure-cli/setup.py

```diff
@@ -129,6 +129,7 @@
     'azure-synapse-artifacts~=0.19.0',
     'azure-synapse-managedprivateendpoints~=0.4.0',
     'azure-synapse-spark~=0.2.0',
+    'cachetools~=5.5.0',
```

Comment:
Would like to get some feedback from the cli team, is it appropriate to add a dependency like this? @zhoxing-ms, @yanzhudd

Reply:
It seems that a global `{sku_name: (cpu_value, nvme_enabled)}` dict can achieve the same thing? Creating a dummy cached function is too heavy.

Reply:
Agreed. Modified the implementation to use a global dict.